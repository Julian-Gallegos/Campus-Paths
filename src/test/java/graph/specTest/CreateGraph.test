# Tests that a graph with one node can be created.

# Create a graph with a node
CreateGraph graph1 node1