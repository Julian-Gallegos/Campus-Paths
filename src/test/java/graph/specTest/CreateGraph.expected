# Tests that a graph with one node can be created.

# Create a graph with a node
created graph1 with node1